src/containers/Board/Board.css

```diff
@@ -42,7 +42,9 @@
 }
 
 .Symbol:focus {
-  outline: 0;
+  border-width: 5px;
+  border-color: blue;
+  border-style: solid;
```

Comment:
How about keeping the `outline: 0` in, because you are kind of replacing it with the border anyway, right? Else we get both the border + the browser-dependent outline.

Also, what do you think about a box-shadow to highlight a focused element with a sort of hover effect? This also would not make the tile contents jump.
Since the app is quite colorful already, I would also suggest a neutral grey color for this.

So, in effect something like this:

```css
.Symbol:focus {
  outline: none;
  box-shadow: 0px 0px 20px 5px #333;
}
```

Reply:
totally agree with the comment

Reply:
Hi. Sorry I was AFK this weekend :-)

Agreed I should have added the outline 0.

Some nitpicks:

The reason I added such a glaring border was that I see the app as an accessibility tool, therefore I would normally accentuate a11y features in such an app. In this case for users with low vision who are not yet using assistive tech to browse.

The box shadow version is a little lower in contrast, but in my opnion not so that it will prove a problem, although I would probably have cranked up the contrast just a little more.

What I do think should be considered is that `border` has better browser support than `box-shadow`. For example, IE 9 upwards supports `box-shadow`, which means that an IE 8 user will experience a totally inaccessible keyboard experience again. Now this is mitigated by the fact that React also only support IE9 upward, but regardless, using `border` insures a larger group of browsers will display this correctly.

Reply:
@AlmeroSteyn 
Yeah, the contrast could be even higher. Agreed :)

You're right regarding the support for `box-shadow`: We could use graceful degradation here. I think there is some "old trick" for IE<9 and the rest would just pick up `box-shadow`. Would have to look it up again though.

Reply:
@dakeyras7 Hey as long as the solution is as widely supported as the browser outline itself, be-it from a fallback or not, it's got my vote :)